cloud-init/copr_build.py

```diff
@@ -120,7 +120,7 @@ def main(srpm, copr_conf=DEFAULT_COPR_CONF, dev=None):
 
     # 2018-03-05: adding sleep to let builds ramp up
     # this after a {'state': ['Not a valid choice.']} exception
-    time.sleep(180)
+    time.sleep(300)
```

Comment:
Is it worth switching to a backoff?
```
for timeout in [60, 120, 240, 360]:
      time.sleep(timeout)
```
or check more frequently:

```
for timeout in range(1, 6):
   time.sleep(60)
```

Reply:
In practice this speed up the job completion only when it fails, as the full build takes more than 5 minutes. But sounds reasonable. I think a condition we can wait wait for is `tasks.keys()` to be non-empty.